napari/layers/_shapes_layer/shape_util.py

```diff
@@ -255,6 +255,29 @@ def orientation(p, q, r):
     return val
 
 
+def is_colinear(points):
+    """Determines is a list of 2D points are colinear
+
+    Parameters
+    -------
+    points : np.ndarray
+        Nx2 array of points to be tested for colinearity
+
+    Returns
+    -------
+    val : bool
+        True is all points are colinear, False otherwise.
+    """
+    if len(points) < 3:
+        return True
+
+    for p in points[2:]:
```

Comment:
what's the `[2:]` for?

Reply:
The collinearity test takes three points, the first two are the first two in the list, and then the third is iterated through in the loop. I will add a comment